python_liftbridge/api_pb2_grpc.py

```diff
@@ -5,102 +5,276 @@
-      ),
-      'Publish': grpc.unary_unary_rpc_method_handler(
-          servicer.Publish,
-          request_deserializer=protos_dot_api__pb2.PublishRequest.FromString,
-          response_serializer=protos_dot_api__pb2.PublishResponse.SerializeToString,
-      ),
-  }
-  generic_handler = grpc.method_handlers_generic_handler(
-      'proto.API', rpc_method_handlers)
-  server.add_generic_rpc_handlers((generic_handler,))
+    rpc_method_handlers = {
+            'CreateStream': grpc.unary_unary_rpc_method_handler(
+                    servicer.CreateStream,
+                    request_deserializer=protos_dot_api__pb2.CreateStreamRequest.FromString,
+                    response_serializer=protos_dot_api__pb2.CreateStreamResponse.SerializeToString,
+            ),
+            'DeleteStream': grpc.unary_unary_rpc_method_handler(
+                    servicer.DeleteStream,
+                    request_deserializer=protos_dot_api__pb2.DeleteStreamRequest.FromString,
+                    response_serializer=protos_dot_api__pb2.DeleteStreamResponse.SerializeToString,
+            ),
+            'PauseStream': grpc.unary_unary_rpc_method_handler(
+                    servicer.PauseStream,
+                    request_deserializer=protos_dot_api__pb2.PauseStreamRequest.FromString,
+                    response_serializer=protos_dot_api__pb2.PauseStreamResponse.SerializeToString,
+            ),
+            'Subscribe': grpc.unary_stream_rpc_method_handler(
+                    servicer.Subscribe,
+                    request_deserializer=protos_dot_api__pb2.SubscribeRequest.FromString,
+                    response_serializer=protos_dot_api__pb2.Message.SerializeToString,
+            ),
+            'FetchMetadata': grpc.unary_unary_rpc_method_handler(
+                    servicer.FetchMetadata,
+                    request_deserializer=protos_dot_api__pb2.FetchMetadataRequest.FromString,
+                    response_serializer=protos_dot_api__pb2.FetchMetadataResponse.SerializeToString,
+            ),
+            'Publish': grpc.unary_unary_rpc_method_handler(
+                    servicer.Publish,
+                    request_deserializer=protos_dot_api__pb2.PublishRequest.FromString,
+                    response_serializer=protos_dot_api__pb2.PublishResponse.SerializeToString,
+            ),
+            'PublishToSubject': grpc.unary_unary_rpc_method_handler(
+                    servicer.PublishToSubject,
+                    request_deserializer=protos_dot_api__pb2.PublishToSubjectRequest.FromString,
+                    response_serializer=protos_dot_api__pb2.PublishToSubjectResponse.SerializeToString,
+            ),
+    }
+    generic_handler = grpc.method_handlers_generic_handler(
+            'proto.API', rpc_method_handlers)
+    server.add_generic_rpc_handlers((generic_handler,))
+
+
+ # This class is part of an EXPERIMENTAL API.
+class API(object):
```

Comment:
Would you mind give some explanation on the specific use case of this class ? What is the purpose of this one ?

Reply:
https://github.com/liftbridge-io/liftbridge-api/blob/v1.0.0/py/api_pb2_grpc.py This file is from the `liftbridge-api`, I didn't remove the code that is not used by the library, but if it is not in use here, I can remove it.